browser-interface/packages/shared/comms/handlers.ts

```diff
@@ -73,68 +70,23 @@ export async function bindHandlersToCommsContext(room: RoomConnection) {
   room.events.on('DISCONNECTION', (event) => handleDisconnectionEvent(event, room))
 }
 
-const pendingProfileRequests: Map<string, Set<IFuture<Avatar | null>>> = new Map()
-
-export async function requestProfileToPeers(
+/**
+ *
+ * @returns true if the user is connected, false if not
```

Comment:
I'm not that sure about this change. If you decide tomove forward with this refactor I'd invest more on reviewing this

Reply:
I strongly believe that this logic should be on `shared/profiles` and not as part of comms handlers...